### ISSUE TITLE
Kong config (yaml) parser demands quoted strings

### ISSUE THREAD

**Issue by grimm26** [Mashape/kong#1277]
Kong freaks out when strings are not quoted in the config:
```
$ head /etc/kong/kong.yml
# Generated by Chef
---
cluster_listen: 0.0.0.0:7946
admin_api_listen: 0.0.0.0:8001
proxy_listen: 0.0.0.0:8000
send_anonymous_reports: false
memory_cache_size: 256
database: postgres
postgres:
  host: 127.0.0.1
....
$ sudo kong start
[INFO] kong 0.8.2
[INFO] Using configuration: /etc/kong/kong.yml
[WARN] admin_api_listen: must be a string
[WARN] postgres.host: must be a string
[WARN] proxy_listen: must be a string
[WARN] cluster_listen: must be a string
[ERR] Invalid properties in given configuration file
```

**Comment by hutchic** [comment]
can you paste the relevant parts of the config?

**Comment by grimm26** [comment]
I did.
```
---
cluster_listen: 0.0.0.0:7946
admin_api_listen: 0.0.0.0:8001
proxy_listen: 0.0.0.0:8000
send_anonymous_reports: false
memory_cache_size: 256
database: postgres
postgres:
  host: 127.0.0.1
```

**Comment by Tieske** [comment]
@thibaultCha This is being refactored already no?

**Comment by thefosk** [comment]
The new configuration file format it's not YAML anymore - so this issue will be closed once `refactor/cli` is merged.

**Comment by Tieske** [comment]
So this is a "won't fix". Let's keep it clean and tidy. Closing it now.

**Comment by gravitystorm** [comment]
It would be good to fix this bug in the 0.7 and 0.8 series. For those of us who generate kong config files (e.g. from chef) this makes life unnecessarily difficult - especially since there is not yet any newer version released.
